integrations/lib/multiwoven/integrations/rollout.rb

```diff
@@ -41,6 +41,7 @@ module Integrations
       Oracle
       MicrosoftExcel
       MicrosoftSql
+      Mailchimp
```

Comment:
@xyfer17 I believe the Version should increase here. @afthabvp Second opinion here

Reply:
@afthabvp can you help me out for the above comment ?

Reply:
@xyfer17 
https://github.com/Multiwoven/multiwoven/blob/93e8767684236e1eea9cb55b6e9719de14afd671/integrations/lib/multiwoven/integrations/rollout.rb#L5
for each new connector need to update version 0.12.0 -> 0.13.0
once this version updated in gem we need to update the server gem

Reply:
@afthabvp have updated the version in commit  https://github.com/Multiwoven/multiwoven/pull/437/commits/af0c76df7fc88f170578433d2d9ab507edccbc83